app/kuma-cp/cmd/cmd_suite_test.go

```diff
@@ -19,21 +19,9 @@ package cmd
 import (
 	"testing"
 
-	"github.com/go-logr/logr"
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-	logf "sigs.k8s.io/controller-runtime/pkg/log"
-	"sigs.k8s.io/controller-runtime/pkg/log/zap"
-
-	"github.com/kumahq/kuma/pkg/core"
+	"github.com/kumahq/kuma/pkg/test"
 )
 
 func TestCmd(t *testing.T) {
-	RegisterFailHandler(Fail)
-	RunSpecs(t, "CMD Suite")
+	test.RunSpecs(t, "CMD Suite")
```

Comment:
If it doesn't take much time can we rename import from `test` to `kuma_test` to make it clear we're using helper function?

Reply:
There's no package name ambiguity here, so I'd prefer to leave it as it is. Maintaining package alias names is just more work :)